CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### `Updated`
+
+- Updated TSeemann's MLST default container. See [PR 97](https://github.com/phac-nml/mikrokondo/pull/97)
```

Comment:
Can you mention which version you update to.

Reply:
Gladly! fixed in: https://github.com/phac-nml/mikrokondo/pull/98/commits/5ac1399fb1bbee0d7e22db2eb5eb052ca384612f

Reply:
Thanks so much.